packages/react-composites/tests/README.md

```diff
@@ -0,0 +1,130 @@
+# Composite browser tests
+
+This folder contains some hermetic and live browser tests for the composites. See the [top-level testing documentation](../../../docs/references/automated-tests.md) to understand how these tests fit into our overall testing strategy.
+
+## Folder structure
+
+Tests for each composite are contained in their own folder. Further, hermetic and live tests for each composite are contained in separate sub-folders.
+
+* [./browser/call](./browser/call) - Tests for `CallComposite`
+  * [./browser/call/app](./browser/call/app) - Test application used for all `CallComposite` tests
+  * [./browser/call/hermetic](./browser/call/hermetic) - Hermetic tests for `CallComposite`
```

Comment:
What's the difference b/w hermetic and fake-adapter?
Would call not have fake-adapter tests as well?

Reply:
fake-adapter is *how* we make sure the test are hermetic. _Hermetic_ better describes what's special about these tests.

For example, the tests could also be made hermetic via a different kind of fake - a full in-memory service implementation, or a record-replay type implementation etc.